misc/python/materialize/cli/mzcompose.py

```diff
@@ -360,30 +360,52 @@ def run(self, args: argparse.Namespace) -> None:
         if not service:
             raise UIError(f"unknown service {args.service!r}")
 
-        image = service["image"].split(":")[0]
-        if image != "materialize/materialized":
-            raise UIError(
-                f"cannot connect SQL shell to non-materialized service {args.service!r}"
-            )
-
         # Attempting to load the default port will produce a nice error message
         # if the service isn't running or isn't exposing a port.
         composition.default_port(args.service)
 
-        deps = composition.repo.resolve_dependencies([composition.repo.images["psql"]])
-        deps.acquire()
-        deps["psql"].run(
-            [
-                "-h",
-                service.get("hostname", args.service),
-                "-p",
-                "6875",
-                "-U",
-                "materialize",
-                "materialize",
-            ],
-            docker_args=["--interactive", f"--network={composition.name}_default"],
-        )
+        image = service["image"].split(":")[0]
```

Comment:
this change is to allow running `./mzcompose sql mysql` in addition to `./mzcompose sql materialized` to get dropped into a SQL Shell connected to mysql/MZ

Reply:
did the old `./mzcompose sql` support postgres containers? I think it did, unless im remembering wrong

Reply:
No it doesn't at the moment, it just supported the MZ container. I'll add the PG support in a follow up PR